Lib/test/test_urlparse.py

```diff
@@ -1053,7 +1054,7 @@ def test_splitport(self):
         self.assertEqual(splitport(':88'), ('', '88'))
 
     def test_splitnport(self):
-        splitnport = urllib.parse.splitnport
+        splitnport = urllib.parse._splitnport
```

Comment:
If you tested the proper function, it would reveal that the new signature is incompatible.